auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -530,6 +528,17 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
     padding: .875rem;
   }
 
+  // size the table according to window height
+  ::v-deep .v-data-table__wrapper {
+    min-height: 9rem; // height of no-data div
+    height: calc(100vh - 31rem); // full height minus footer
+    background-color: $gray1; // hide unused space below table
+
+    table {
+      background-color: white;
+    }
+  }
```

Comment:
@cameron-freshworks FYI, little trick for better space management :)

Reply:
@severinbeauvais Very cool!

Reply:
Cause between you and me, we see a lot of tables 😆

Reply:
We do indeed 😆